api/app/src/Repository/DeputyRepository.php

```diff
@@ -44,16 +44,27 @@ public function getUidToIdMapping(): array
     public function findReportsInfoByUid(int $uid, bool $includeInactive = false): ?array
     {
         $sql = <<<SQL
-        SELECT c.firstname AS "firstName",
-               c.lastname AS "lastName",
-               c.case_number AS "caseNumber",
-               co.court_order_uid AS "courtOrderUid",
-               r.type AS "type"
-        FROM deputy d
-        LEFT JOIN client c ON c.deputy_id = d.id
-        LEFT JOIN court_order co ON co.client_id = c.id
-        LEFT JOIN report r ON r.client_id = c.id
-        LEFT JOIN court_order_deputy cod ON cod.deputy_id = d.id
+        SELECT DISTINCT
+        c.firstname AS "firstName",
+        c.lastname AS "lastName",
+        c.case_number AS "caseNumber",
+        (
+            SELECT DISTINCT string_agg(co.court_order_uid, ', ')
```

Comment:
If we're doing the concatenating in this SQL, we should order by court order UID ASC, otherwise it might look a bit odd in the UI. I'd suggest at least calling the field "court order UIDs".

Also, aren't we going to be relying on these to link to the next page? If so, it would be nicer if they were an array when returned by the API, so we can easily create the links by loop.

Reply:
> It's my understanding we're going to directly link to the report from this page & not the court order itself.
> 
> The only reason I've had to include the concat is due to hybrid's which have two court orders associated with each report, which I failed to consider when initially doing this ticket & happened across incidentally while testing [DDLS-576](https://opgtransform.atlassian.net/browse/DDLS-576)

If we're going to be linking to the report, I think we're going to need the report ID in the response.